paypal/__init__.py

```diff
@@ -1 +1 @@
-VERSION = '1.0.0'
+VERSION = '1.1.0'
```

Comment:
Could you please leave the version unchanged in this PR? We'll bump it before cutting a new release. I'd like to drop Python 2 support as well in the next release, and then publish a new major version.